src/popover/popover.ts

```diff
@@ -88,13 +95,33 @@ export class NgbPopoverWindow {
     this._renderer.addClass(this._element.nativeElement, 'bs-popover-' + this.placement.toString().split('-')[0]);
     this._renderer.addClass(this._element.nativeElement, 'bs-popover-' + this.placement.toString());
   }
+
+  /**
+   * Tells whether the event has been triggered from this component's subtree or not.
+   *
+   * @param event the event to check
+   *
+   * @return whether the event has been triggered from this component's subtree or not.
+   */
+  isEventFrom(event: Event): boolean { return this._element.nativeElement.contains(event.target as HTMLElement); }
 }
 
 /**
  * A lightweight, extensible directive for fancy popover creation.
  */
 @Directive({selector: '[ngbPopover]', exportAs: 'ngbPopover'})
 export class NgbPopover implements OnInit, OnDestroy, OnChanges {
+  /**
+   * Indicates that popover should be closed on clicks inside the popover,
```

Comment:
Way too long and confusing to me. Maybe `Indicates whether the popover should be closed on Escape key and inside/outside clicks.` ?

Reply:
Agreed, I never liked this sentence 😆